build.gradle

```diff
@@ -120,7 +120,7 @@ if (file('.git').exists()) {
   rat {
     // Exclude everything under the directory that git should be ignoring via .gitignore or that isn't checked in. These
     // restrict us only to files that are checked in or are staged.
-    def repo = Grgit.open(project.getRootDir())
+    def repo = Grgit.open(currentDir: file('.'))
```

Comment:
I don't think this is right, it seems like it assumes you are running the build command from the project root dir. Did the gradle update remove all ability to get the project root directory or did it just move it? In any case, seems surprising they would do that in a minor update...

Reply:
Fair point, I will address this.

Reply:
Done (via recent changes); conversation resolved.